aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -303,6 +305,47 @@ The following sample `aspNetCore` element configures stdout logging for an app h
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-2.2"
+
+## Enhanced diagnostic logs
+
+The ASP.NET Core Module provides is configurable to provide enhanced diagnostics logs. Add the `<handlerSettings>` element to the `<aspNetCore>` element in *web.config*. Setting the `debugLevel` to `TRACE` exposes a higher fidelity of diagnostic information. The example below is configured for an Azure Apps deployment:
+
+```xml
+<aspNetCore processPath="dotnet"
+    arguments=".\MyApp.dll"
+    stdoutLogEnabled="true"
+    stdoutLogFile="\\?\%home%\LogFiles\stdout"
+    hostingModel="inprocess">
+  <handlerSettings>
+    <handlerSetting name="debugFile" value="%HOME%\LogFiles\Application\amcm.log" />
+    <handlerSetting name="debugLevel" value="TRACE" />
+  </handlerSettings>
+</aspNetCore>
+```
+
+Debug level (`debugLevel`) values:
+
+* ERROR
+* WARNING
+* INFO
+* TRACE
+* CONSOLE
+* FILE
+* EVENTLOG
+
+The handler settings can also be provided via environment variables:
+
+* `ASPNETCORE_MODULE_DEBUG_FILE` &ndash; Path to the debug log file.
+  * IIS default: *aspnetcore-debug.log*
```

Comment:
Shouldn't the file names be the same here?

Reply:
I thought I saw that it defaults differently. I'll go back into the repo and see where I saw these and let you know.

Reply:
Found it ... it came from here ...

https://github.com/aspnet/AzureIntegration/blob/master/extensions/Microsoft.AspNetCore.Runtime.SiteExtension/applicationHost.xdt

Reply:
Is that only for the site extension? ... looks that way.

Reply:
This is only for the site extension. This isn't inserted by default for azure deployments. @pakrym to confirm.

Reply:
Aka it will still be *aspnetcore-debug.log* in azure.

Reply:
I'll stand-by on this one until we hear back. If it's set for the site extension as ...

```
%HOME%\LogFiles\Application\amcm.log
```
... and stdout logs to go to ...

```
\\?\%home%\LogFiles\
```
... there might be some lack of consistency in providing no path (the root of the deployment) by just showing `aspnetcore-debug.log`.